src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="product-cards">
+
+    <div
```

Comment:
```suggestion
  <body class="product-cards">
    <div
```
Don't add empty lines between parent and child elements

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/133872866/54ccfe04-934e-487c-a847-15fe2390fb9c)Тут я вніс зміни ще попереднього разу незнаю чому не відображаються